### ISSUE TITLE
Make setting for how many chunks should be created for every file

### ISSUE THREAD

**Issue by henrydatei** [pyload/pyload#4423]
### Feature description
For what I noticed Pyload creates for every file 3 chunks and downloads every file with 3 connections to speed up the download. I would love to see a feature where I can change how many (or even if) chunks I want per file. 

### Problem description
The reason why I want this feature is that my multihoster often throws a 416 request range not satisfiable and I always fix this by deleting all chunks and start the download again. I thinks that if Pyload doesn't create chunks the download would be more stable.

**Comment by GammaC0de** [comment]
> my multihoster often throws a 416 request range not satisfiable

Which multihoster are you referring to?

**Comment by henrydatei** [comment]
Debrid-link.frAm 07.01.2024 um 07:54 schrieb GammaC0de ***@***.***>:﻿

my multihoster often throws a 416 request range not satisfiable

Which multihoster are you referring to?

**Comment by GammaC0de** [comment]
This is weird because when using DebridLinkFr,the chunk limit sent directly from the multihoster (see https://github.com/pyload/pyload/blob/4159a1191ec4fe6d927e57a9c4bb8f54e16c381d/src/pyload/plugins/downloaders/DebridlinkFr.py#L74)

Do you get this issue when using a rapidgator url?

**Comment by henrydatei** [comment]
Yes, but I mostly download RG links with debridLinkFr since they are mostly working. But I don't get the 416 error when downloading directly via the browser from DebridLinkFr, that's why I assume it's a problem with Pyload. 

I used to download DDL through debridLinkFr but quite often the download is corrupted and I have no possibility to check which part is corrupted. With RG links this is easy beachside RG provides the MD5 hash of a file but DDL does not. That's why I now mostly use RG. But it could be that DDL links sometimes throw a 416 when downloading through DebridLink and pyload.
